catalog/core/catalog-core-api/src/main/java/ddf/catalog/transform/Transformer.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transform;
+
+import java.util.Map;
+import java.util.Set;
+import javax.activation.MimeType;
+
+public interface Transformer {
```

Comment:
It seems weird to call this `Transformer` when there aren't any `transform*()` methods on it.

Reply:
I'm open to a different name. The interface represents values that were previously defined as service-properties in blueprint, but are not first class citizens of the transformer.

Reply:
Maybe `TransformerProperties`.

Reply:
done